MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+# SPIRE Helm Charts Maintainership Guidelines and Processes
+
+This document captures the values, guidelines, and processes that the SPIRE Helm Charts project and its maintainers adhere to. All maintainers agree to uphold and abide by the text contained herein.
+
+For a list of active SPIRE Helm Charts maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General governance
+
+The SPIRE Helm CHarts project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the [SPIFFE Steering Committee][3] (SSC) the same way that the SPIFFE project and associated maintainers do.
+
+SSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement.
+
+## Maintainer responsibility
+
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+
```

Comment:
With so few maintiners, https://github.com/spiffe/spiffe/blob/main/GOVERNANCE.md#change-review-process will be hard to accomplish. Especially the 2 maintainer requirement.

Reply:
Here's three that haven't gotten an initial review, just off the bat, as of this writing:
https://github.com/spiffe/helm-charts/pull/209 is at 2 days
https://github.com/spiffe/helm-charts/pull/206 is at 3 days
https://github.com/spiffe/helm-charts/pull/196 is over a week

I'm sure I could dig up other examples, but its probably plenty.

I'm not saying we must be reviewing faster. It would be nice, but with so few revisers, I know its really hard to actually accomplish. But we gotta be careful making rules we can't seem to meet.

Reply:
@kfox1111 There are currently 18 PRs open on this repo, and 15 of them are yours. I appreciate the enthusiasm but Im struggling to keep up with that volume.

Ill add that I think the 24 hour rule mostly applies to community contributed PRs and not so much between maintainers.

Reply:
Yes, we're having a hard time with reviews. I've slowed way down in submitting PR's as the existing ones are taking time and work to keep going, let alone introducing many new ones.

But "slow down" is not necessarily the right answer to the problem either, if the goal is to get a stable and usable chart soon.

If we're coming up with rules, people will try and play by them. If faster reviews happens by being a contributor and not a maintainer, that just discourages folks from being maintainers.... Something I don't think is a good idea...